docs/EVG.md

```diff
@@ -617,77 +585,81 @@ The requirements in Section 7.1 of the Baseline Requirements apply equally to EV
 
-(2)  The following fields MUST be present if the Subordinate CA is not controlled by the entity that controls the Root CA.
+   Otherwise, it MAY contain the anyPolicy identifier.
 
-   certificatePolicies:policyQualifiers:policyQualifierId
+2. The following fields MUST be present if the Subordinate CA is not controlled by the entity that controls the Root CA.
 
-    - id-qt 1 [RFC 5280]
+   * `certificatePolicies:policyQualifiers:policyQualifierId`
 
-   certificatePolicies:policyQualifiers:qualifier:cPSuri
+      id-qt 1 [RFC 5280]
 
-    - HTTP URL for the Root CA's Certification Practice Statement
+   * `certificatePolicies:policyQualifiers:qualifier:cPSuri`
 
-(3) The certificatePolicies extension in EV Certificates issued to Subscribers MUST include the following:
+      HTTP URL for the Root CA's Certification Practice Statement
 
-   certificatePolicies:policyIdentifier (Required)
+3. The certificatePolicies extension in EV Certificates issued to Subscribers MUST include the following:
 
-    - The Issuer's EV policy identifier
+   * `certificatePolicies:policyIdentifier` (Required)
 
-   certificatePolicies:policyQualifiers:policyQualifierId (Required)
+      The Issuer's EV policy identifier
 
-    - id-qt 1 [RFC 5280]
+   * `certificatePolicies:policyQualifiers:policyQualifierId` (Required)
+      
+      id-qt 1 [RFC 5280]
 
-   certificatePolicies:policyQualifiers:qualifier:cPSuri (Required)
+   * `certificatePolicies:policyQualifiers:qualifier:cPSuri` (Required)
 
-    - HTTP URL for the Subordinate CA's Certification Practice Statement
+      HTTP URL for the Subordinate CA's Certification Practice Statement
 
-(4)  The cRLDistributionPoints extension MUST be present in Subscriber Certificates if the certificate does not specify OCSP responder locations in an authorityInformationAccess extension.
+4. The cRLDistributionPoints extension MUST be present in Subscriber Certificates if the certificate does not specify OCSP responder locations in an authorityInformationAccess extension.
 
 ## 9.8. Certificate Extensions
 
-The extensions listed in the Section 9.8 are recommended for maximum interoperability between certificates and browsers / applications, but are not mandatory on the CAs except where indicated as “Required”.  CAs may use other extensions that are not listed in this Section 9.8, but are encouraged to add them to this section by ballot from time to time to help increase extension standardization across the industry.  
+The extensions listed in this [Section 9.8](#certificate-extensions) are recommended for maximum interoperability between certificates and browsers / applications, but are not mandatory on the CAs except where indicated as “Required”.  CAs may use other extensions that are not listed in this [Section 9.8](#certificate-extensions), but are encouraged to add them to this section by ballot from time to time to help increase extension standardization across the industry.
```

Comment:
I intentionally didn't clean up all the references to Sections from a grammar point, because we are wildly inconsistent and many of them don't make sense.

I wanted to minimize some of that churn, other than keeping this symmetrical between the first sentence and the second sentence.

Reply:
That's a reasonable point of view, but maybe we should fix them while were here?  I could review them and submit a pull request if you'd like.

Reply:
Looks like [our comments](https://github.com/cabforum/servercert/pull/235#issuecomment-738965945) collided. Certainly not opposed, and you should be able to search for `Section \d` now (previously, the query was `[sS]ection(s)? \d` - I just wanted to minimize churn for folks and get initial reactions :)